__tests__/app.test.js

```diff
@@ -472,5 +472,26 @@ describe('GET /api/articles to query', () => {
 
 })
 
+describe('GET /api/articles/:article_id comment_count as a query', () => {
+    test('200: get the comment count is asking for it', () => {
+        return request(app)
+        .get('/api/articles/3?add_on=comment_count')
```

Comment:
You don't need to add this as a query, the endpoint should just be api/articles/3